moto/ram/models.py

```diff
@@ -413,5 +346,49 @@ def list_resource_types(self, **kwargs: Any) -> Dict[str, Any]:
 
         return dict(resourceTypes=resource_types)
 
+    def list_permissions(
+        self, resource_type: str, permission_type: str
+    ) -> Dict[str, Any]:
+        permission_types_relation = {
+            "AWS": "AWS_MANAGED",
+            "LOCAL": "CUSTOMER_MANAGED",
+        }
+
+        # Here, resourceType first partition (service) is case sensitive and
+        # last partition (type) is case insensitive
+        if resource_type and not any(
+            str(permission["resourceType"]).split(":")[0] == resource_type.split(":")[0]
+            and str(permission["resourceType"]).split(":")[-1].lower()
+            == resource_type.split(":")[-1].lower()
+            for permission in self.MANAGED_PERMISSIONS
+        ):
```

Comment:
It took me a while to see what's going on in this logic, even with the comment. 

What do you think about this simplification?
 - The `str(..)` should not be necessary, as we know it's already a string
 - We can compare the entire string for make it more readable. If the entire string is equal, then the last partition is also equal

```suggestion
        if resource_type and not any(
            permission["resourceType"].split(":")[0] == resource_type.split(":")[0]
            and permission["resourceType"].lower() == resource_type.lower()
            for permission in self.MANAGED_PERMISSIONS
        ):
```
Note that this is just a suggestion, nothing else! Feel free to push back if you prefer your version.